health/health.go

```diff
@@ -170,6 +170,20 @@ func Register(name string, check Checker) {
 	registeredChecks[name] = check
 }
 
+// Unregister removes the named checker.
+func Unregister(name string) {
```

Comment:
I'm not a fan of `Unregister`. This pattern doesn't exist in the stdlib. Can we test the health checkers without registration?